helix-admin-webapp/src/main/java/org/apache/helix/webapp/resources/ResourceUtil.java

```diff
@@ -45,9 +45,9 @@
 
 public class ResourceUtil {
   private static final Logger LOG = LoggerFactory.getLogger(ResourceUtil.class);
+  private static ObjectMapper mapper = new ObjectMapper();
   private static final String EMPTY_ZNRECORD_STRING =
       objectToJson(ClusterRepresentationUtil.EMPTY_ZNRECORD);
```

Comment:
Thanks for the contribution.
I think you can safely remove `EMPTY_ZNRECORD_STRING` and `objectToJson`, I don't think they are ever used.

Reply:
I'll try it out.

Reply:
Was able to remove and added a `final` to the mapper (per Intellij's recommendations)